src/Modal/ModalPortal.tsx

```diff
@@ -1,10 +1,23 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import ReactTransitionGroup from 'react-transition-group/TransitionGroup';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as ReactTransitionGroup from 'react-transition-group/TransitionGroup';
 import { props, t, ReactChildren } from '../utils';
 import TransitionWrapper from '../TransitionWrapper';
 
-let containerNode = null;
```

Comment:
unrelated to this PR but weird.. why are we allowing a single `containerNode` in any app? :thinking: /cc @FrancescoCioria maybe worth opening a separate issue